.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+- nightly-2016-04-11
+before_script:
+- |
+  pip install 'travis-cargo<0.2' --user &&
```

Comment:
Alright!

Oh, and sorry for the typos in that last comment 😄
Scott Olson <notifications@github.com> schrieb am Fr., 15. Apr. 2016 um
08:41:

> In .travis.yml <https://github.com/tsion/miri/pull/3#discussion_r59832429>
> :
>
> > @@ -0,0 +1,14 @@
> > +language: rust
> > +rust:
> > +- nightly-2016-04-11
> > +before_script:
> > +- |
> > +  pip install 'travis-cargo<0.2' --user &&
>
> Should I add --nocapture so we can see the actual test output even if
> there was no failure?
>
> Yeah, that sounds good for now.
>
> So you want to hate the build on this or should it be an allowed failure?
>
> I think allowed failure is fine.
>
> So my understanding is that the script as currently written doesn't
> benefit from travis-cargo but we could make use of it in the future. I'm
> fine with that.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tsion/miri/pull/3/files/99767e7a293c2ea26725a047fcc870af97d58ee3#r59832429>
>